### ISSUE TITLE
Performance on ADE20K

### ISSUE THREAD

**Issue by hubertlee915** [PkuRainBow/OCNet.pytorch#53]
Hi! Could you indicate which specific model of OCNet (BaseOC, Pyramid-OC, or ASP-OC) obtain 45.45% mIoU on ADE20K validation set? Thank you!

**Comment by PkuRainBow** [comment]
ResNet-101 + ASP-OC

**Comment by hubertlee915** [comment]
Thank you!

**Comment by hubertlee915** [comment]
Could you provide the code and trained model on ADE20K dataset with ResNet-101 + ASP-OC model? Thank you!

**Comment by hubertlee915** [comment]
Could you provide the code and trained model on ADE20K dataset with ResNet-101 + ASP-OC model? Thank you!
